password/src/Data/Password/Bcrypt.hs

```diff
@@ -183,6 +187,22 @@ checkPassword pass (PasswordHash passHash) =
       then PasswordCheckSuccess
       else PasswordCheckFail
 
+-- | Extracts the cost parameter as an 'Int' from a 'PasswordHash' 'Bcrypt'
+--
+-- >>> let pass = mkPassword "foobar"
+-- >>> passHash <- hashPassword pass
+-- >>> extractParams passHash == Just 10
+-- True
+--
+-- @since 3.0.2.0
+extractParams :: PasswordHash Bcrypt -> Maybe Int
+extractParams (PasswordHash passHash) =
+  case T.split (== '$') passHash of
+    [_, version, cost, _pass] -> do
+      guard $ elem version $ map T.pack ["2", " 2a", " 2x", " 2y", " 2b"]
```

Comment:
Found the problem.

These strings have a space in front of them :/

Reply:
--' sorry, everything is green now, thanks a lot

Reply:
Hey, I also didn't notice when reviewing :sweat_smile:

Reply:
Yes, but I did the copy and paste, that's called the karma :)